test/functional/pdo_sqlsrv/pdo_092_emulate_prepare_statement_utf8.phpt

```diff
@@ -1,65 +1,69 @@
 --TEST--
 Test emulate prepare utf8 encoding set at the statement level
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-require_once("MsSetup.inc");
+require_once("MsCommon_mid-refactor.inc");
 
-$pdo_options = [];
-$pdo_options[PDO::ATTR_ERRMODE] = PDO::ERRMODE_EXCEPTION;
-$pdo_options[PDO::SQLSRV_ATTR_ENCODING] = PDO::SQLSRV_ENCODING_UTF8;
-
-$connection = new PDO("sqlsrv:server=$server;Database=$databaseName",  $uid, $pwd, $pdo_options);
+try {
+    $pdo_options = [];
+    $pdo_options[PDO::ATTR_ERRMODE] = PDO::ERRMODE_EXCEPTION;
+    $pdo_options[PDO::SQLSRV_ATTR_ENCODING] = PDO::SQLSRV_ENCODING_UTF8;
 
-$pdo_options = array();
-$pdo_options[PDO::ATTR_EMULATE_PREPARES] = TRUE;
-$pdo_options[PDO::SQLSRV_ATTR_DIRECT_QUERY] = TRUE;
-$pdo_options[PDO::SQLSRV_ATTR_ENCODING] = PDO::SQLSRV_ENCODING_UTF8;
-$pdo_options[PDO::ATTR_CURSOR] = PDO::CURSOR_SCROLL;
-$pdo_options[PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE] = PDO::SQLSRV_CURSOR_BUFFERED;
+    $connection = connect('', $pdo_options);
 
-// Drop
-try {
-    $st = $connection->prepare("DROP TABLE TEST", $pdo_options);
-    $st->execute();
-}
-catch(\Exception $e) {}
+    // Always Encrypted does not support using DIRECT_QUERY for binding parameters
+    $pdo_options = [];
```

Comment:
Again, is there a reference page that supports your reason of not using DIRECT_QUERY?

Reply:
In fact, we probably can skip this test by not using any AE features because with AE, the main essence of this test is lost
The same applies to the other test pdo_138*

Reply:
There's no reference for this since it's related to our code. In short it's the same reason as not supporting parameterized query in SQLSRV. I'll add a wiki page for this later.

I understand with AE we're not testing emulate prepare in the test, but I think it's nice to have it to test the utf8 part